src/main/java/Demo.java

```diff
@@ -1,7 +1,100 @@
 package main.java;
 
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Demo {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        calculator1();
+    }
+
+    public static void calculator1() {
+        Scanner sc = new Scanner(System.in);
```

Comment:
Scanner should not be a part of your calculator. How input gets converted to what it is should come from the outside.